### PR TITLE
test: Tabs

### PR DESCRIPTION
**Pull request by yardz**
### SUMMARY
Test for Tabs component

### CODE REVIEW

File: superset-frontend/src/dashboard/components/gridComponents/Tabs.test.tsx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        <button type="button" data-test="DragDroppable" onClick={props.onDrop}>
+          DragDroppable
+        </button>
+        {props.children(childProps)}
+      </div>
+    );
+  }),
+);
+
+const creteProps = () => ({
```
Comment (pkdotson) [inline superset-frontend/src/dashboard/components/gridComponents/Tabs.test.tsx:76]: spelling nit: createProps
Also why does this need to be a function. There are not paramerters being passed in or reassignments. Does it make sense to make this an object.

Comment (yardz) [reply superset-frontend/src/dashboard/components/gridComponents/Tabs.test.tsx]: Fixed!

We talked about this in another PR: https://github.com/apache/superset/pull/13753#discussion_r600862661

File: superset-frontend/src/dashboard/components/gridComponents/Tabs.test.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  });
+
+  expect(props.deleteComponent).not.toBeCalled();
+  expect(screen.queryByText('Delete dashboard tab?')).not.toBeInTheDocument();
+  userEvent.click(screen.getAllByRole('button', { name: 'remove' })[0]);
+  expect(props.deleteComponent).not.toBeCalled();
+
+  await waitFor(() => {
+    expect(screen.getByText('Delete dashboard tab?')).toBeInTheDocument();
+  });
```
Comment (michael-s-molina) [inline superset-frontend/src/dashboard/components/gridComponents/Tabs.test.tsx:199]:
```suggestion
  expect(await screen.findByText('Delete dashboard tab?')).toBeInTheDocument();
```

**Review by pkdotson — APPROVED** [review]
LGTG!

**Pull request merged**